packages/text-bitmap/test/BitmapFont.js

```diff
@@ -35,16 +56,9 @@ describe('BitmapFont', function ()
             expect(fontRes2.chars[id].texture.baseTexture.resolution).to.equal(2);
         });
 
-        it('should throw an error when name collision is detected in bitmap font registry', function ()
```

Comment:
Maybe we could change this test to detect that the font was "replaced" instead of deleting it.

Reply:
Done.